test/arbitrator.js

```diff
@@ -100,33 +101,69 @@ contract("HatVaults Arbitrator", (accounts) => {
     const claimId = await submitClaim(hatVaults, { accounts });
 
     // challengeClaim will fail if passing an non-existent claimID
-    assertFunctionRaisesException(
+    await assertFunctionRaisesException(
       hatVaults.challengeClaim("1234", { from: accounts[2] }),
-      "HVE10"
+      "NoActiveClaimExists"
```

Comment:
I eam not getting this. Did the test pass before? If so, is the test even working?

Reply:
It was missing `await` in the `assertFunctionRaisesException` call so it wasn't working before, other calls missed it too I think I fixed them all though

Reply:
Good catch :)